pkg/apis/cluster/v1alpha1/cluster_types_test.go

```diff
@@ -36,24 +36,39 @@ func TestStorageCluster(t *testing.T) {
 			},
 		},
 	}
-	g := gomega.NewGomegaWithT(t)
 
 	// Test Create
 	fetched := &Cluster{}
-	g.Expect(c.Create(context.TODO(), created)).NotTo(gomega.HaveOccurred())
+	if err := c.Create(context.TODO(), created); err != nil {
+		t.Errorf("error creating cluster: %v", err)
```

Comment:
I didn't mention it earlier, but strings passed to `t.Errorf` or `t.Fatalf` should start with a capital letter, as they are line-oriented strings (as opposed to error strings which start with a lower case letter). Can you please fix across the PR?